capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -118,6 +127,11 @@ function keepTrackOfChosenColleges(validSuggestion) {
   const listElement = document.createElement('li');
   listElement.setAttribute('id', validSuggestion.collegeId.id);
   listElement.innerHTML = validSuggestion.collegeName;
+  listElement.innerHTML += ' ';
```

Comment:
```suggestion
  listElement.innerHTML += '<br> ';
```
so that it looks like this:
![image](https://user-images.githubusercontent.com/43079005/91916471-1589ca00-ec72-11ea-9119-2071bd77267d.png)